src/main/java/uk/yermak/audiobookconverter/fx/FilesController.java

```diff
@@ -150,6 +150,17 @@ public void initialize() {
         filesChapters.getTabs().remove(chaptersTab);
 
         bookStructure.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
+        bookStructure.getSelectionModel().getSelectedItems().addListener((ListChangeListener<TreeItem<Organisable>>) c -> {
+            List<MediaInfo> list = ConverterApplication.getContext().getSelectedMedia();
+            list.clear();
+            for (TreeItem<Organisable> item : c.getList()) {
```

Comment:
Is it better solution to extend interface Organisable with method getMedia and avoid instance check completely?

Reply:
Replaced with Organizable.getMedia()